config/app/variables/cli/cli.json

```diff
@@ -65,7 +65,9 @@
         }
       },
       "options": {
-        "usage": "node ./bin/server.js [args]"
+        "usage": "node ./bin/server.js [args]",
+        "envVarPrefix": "CSS",
+        "loadFromEnv": true
```

Comment:
Spent way too long looking for these parameters in the yargs documentation until I realized they were ours.